http/oidc/pom.xml

```diff
@@ -31,7 +31,6 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>wildfly-elytron-http-oidc</artifactId>
-
```

Comment:
Thanks very much for the updates, @fazer1929! Just one more very minor comment - it's good to avoid whitespace changes like this and below.